viewing_party/main.py

```diff
@@ -0,0 +1,157 @@
+def create_movie(movie_title,genre,rating):
+
+    new_movie={
+        "title":movie_title,
+        "genre":genre,
+        "rating":rating
+    }
+
+    for values in new_movie.values():
+      if values==None:
+        return None
+      
+    return new_movie
+
+def add_to_watched(user_data, movie):
+    added_data = user_data["watched"] 
+    added_data.append(movie) 
+    return user_data
+    
+
+def add_to_watchlist(user_data, movie):
+    added_data = user_data["watchlist"] 
+    added_data.append(movie) 
+    return user_data   
+
+
+def watch_movie(user_data, title):
+    user_watchlist_list=user_data["watchlist"]
+    
+    for dicts in user_watchlist_list:
+        
+        if dicts["title"]==title:
+        
+            user_data=add_to_watched(user_data,dicts)
+            
+            user_data["watchlist"].remove(dicts)
+            return user_data
+        
+    return user_data
+       
+
+
+def get_watched_avg_rating(user_data):
+    total_rating=0
+    
+    user_watched_list=user_data["watched"]
+    if user_watched_list==[]:
+      return float(0)
+    else:
+      for item in user_watched_list:
+      
+        total_rating+=item["rating"]
+        average=total_rating/len(user_watched_list)
+      return average
```

Comment:
this works great! but let's try using only one `return` statement:

```suggestion
    total_rating=0.0
    if len(user_data["watched"] > 0:
        for item in user_watched_list:
            total_rating+=item["rating"]
        total_rating=total_rating/len(user_data["watched"])
    return total_rating
```